deny.toml

```diff
@@ -231,6 +231,13 @@ ignore = [
     "RUSTSEC-2023-0086",
     # Use standard library's IsTerminal trait instead of `atty` (unmaintained)
     "RUSTSEC-2024-0375",
+    # `derivative` is unmaintained; consider using an alternative (unmaintained)
+    "RUSTSEC-2024-0388",
+    # Multiple soundness issues
+    "RUSTSEC-2024-0379",
+    # `instant` is unmaintained, and the author recommends using the maintained [`web-time`] crate instead.
+    "RUSTSEC-2024-0384",
```

Comment:
Should we switch to the alternatives instead of ignoring these?

Reply:
Yes, definitely, but I didn't want to do it myself, I consider this list to be a big TODO basically

Reply:
I guess my question is, why even add these exceptions in the first place? Is it breaking CI or something?

Reply:
Yes, we have a "security" pipeline which will ping me every day about this if I don't silence it: https://buildkite.com/materialize/security

Reply:
Looking at the commit history of this file, it doesn't look like we have a great track record of actually resolving these issues and removing the ignores. I don't mean to throw a wrench in this and I'm not at all familiar with the process but it sounds like something might be slightly broken with the process.

Maybe the security pings should be going somewhere else, but if our current process is to silence the pings without resolving them, then it doesn't sound like they're that useful in the first place.

Reply:
"Multiple soundness issues" for float parsing is a bit scary, I put up https://github.com/MaterializeInc/materialize/pull/30426 to remove our use. Otherwise IMO the "unmaintained" issues aren't great but aren't too bad either

Reply:
Whoops sorry I didn't realize auto merge was enabled before I accepted the PR. If possible @def- would love your help in getting https://github.com/MaterializeInc/materialize/pull/30426 merged